web/src/routes/PoisPage.tsx

```diff
@@ -56,35 +56,25 @@ const moveViewToBBox = (bBox: BBox, defaultVp: MapViewMercatorViewport): MapView
 
 const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: CityRouteProps): ReactElement => {
   const { urlSlug } = useParams()
+  const normalizedUrlSlug = urlSlug ? normalizePath(urlSlug.toLowerCase()) : undefined
```

Comment:
I think we should include the `toLowerCase` to normalizePath. We don't have case-sensitive urls.

Reply:
```suggestion
  const normalizedUrlSlug = urlSlug ? normalizePath(urlSlug) : undefined
```